test/run-drun/max-stack.mo

```diff
@@ -2,7 +2,7 @@
 import { errorMessage; debugPrint; } = "mo:⛔";
 
 actor {
-    let expectedMinimumSize = 31_000;
+    let expectedMinimumSize = 23_000;
```

Comment:
Maybe that's the type table bug I fixed and can be reverted?

Reply:
Indeed. I reverted this change. Thanks.